### ISSUE TITLE
Issues with v2+

### ISSUE THREAD

**Issue by Warchas** [Roeshambo/MonolithDKP#172]
So I did the migration of our old tables to the new. (v2.0.3)
Then you posted a new version 2.0.4

Didn't break or anything.

We then did 2 Onyxia runs

It seem to update the table properly, we got one group added and the other added aswell.

Then we did an MC trash run.

After all was done everyone had proper tables.

Now today I logged on at my secondary computer the addon was updated to v2.0.5.

When I logged on my officers got a message saying 

"MonolithDKP: Invalid entries were removed. Please validate your tables. (In the right click context menu on the DKP table)"

and the stuff that was added that I didnt have on my secondary computer were all removed.

I then started my main computer and copied the files from our run yesterday over to the secondary computer and started wow again.

Now when I logged on I got the same message on my alt character (also an officer).

I am going to try to put the files on my main (Guildmaster and the one that did the migration and see what the AddOn does).

This is a big issue if this is how it works though.

**Comment by Warchas** [comment]
And now when I did it with the guildmaster it updated properly.

This really stupid cause then officers can not do raids without the GM online cause when the GM logs on the entries they made will be deleted.

**Comment by Warchas** [comment]
TL;DR

If an officer adds something to the DKP AddOn while Guildmaster or infact probably the one that is in the note of the GM {MonDKP=Warchas} is offline. 
If then whoever is the one in the "{MonDKP=Warchas}" logs back online it will tell them that they have invalid entries and delete the entries that they made meaning you will never be able to new players to the dkp table, fix issues or raid without whoever is in "{MonDKP=Warchas}".


Big issue, this was what I was asking you about in another issue, how will the addon decide on what is correct or not when you dont have a timestamp in the note anymore?

A problem I foresee even if this is fixed.

Lets say it counts entries by numbers;

And when I am onlnie there is 100 entries.

I logout and then an officer adds an adjustment making it 101 entries.

Then he logs out and I log back in again, for me there is still 100 entries, and then if I add another entries making it 101 for me aswell. *

What happens when he logs back in?

If it goes by entries then both have 101 but it's not correct because of that. 

See the issue I am trying to convey?

**Comment by Roeshambo** [comment]
It doesn't count entries by just number. It counts entries by Officer and number. The entries it's deleting are considered invalid because they are created in your name and you did not create them. There may be a bug here. But when you sync, you tell everyone what you have created. Say you've made 100 entries. Your number would be 100% accurate because you made them. So if someone, for whatever reason has 104 entries made by you, the 4 extra are invalid and are removed. The GM note is completely irrelevant. You can delete that as it's not used within the addon. Please post your Saved Variables file and we'll see what might be hanging up. It was almost certainly something that got jumbled in the migration so we'll just get that straightened out and you'll be all set

**Comment by Roeshambo** [comment]
To clarify, all indexes are created with officer-#. So You can have 100 entries, and another officer can have 50 entries while you're offline. You will still go up to OfficerA-100, while the other officer has OfficerB-50.

**Comment by Roeshambo** [comment]
Looking this over, I believe I know exactly what happened. When you sync, you tell everyone how many entries you yourself have created. As your information would be correct since you were the one that crated them. Any excess entries someone has are removed. I believe when you logged into the secondary computer, it did not have your new entries on it so you corrected that. Say you created entries 201 through 215 on your primary computer. Then log into your secondary computer and your Meta table says you're still on 200. You tell everyone they should also be on 200 and to erase entries beyond that. I honestly had not even thought about logging in from a separate machine. I believe it would be remedied by logging into an alt within the guild on your secondary computer in that event to download that new information from officers and then you'd be fine on your main. Or you could simply copy the SavedVariables file from the primary computer to the secondary. I'll look into a possible fix for that.
